### ISSUE TITLE
Component error on start (NameError: name 'user_input' is not defined)

### ISSUE THREAD

**Issue by lweberru** [mammuth/ha-fritzbox-tools#63]
I installed the component via HACS
and configured my fritzbox as mentioned in the docs. 
But component is not beeing discovered, getting following error message in the log 

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 109, in async_init
    result = await self._async_handle_step(flow, flow.init_step, data)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 172, in _async_handle_step
    result: Dict = await getattr(flow, method)(user_input)
  File "/config/custom_components/fritzbox_tools/config_flow.py", line 170, in async_step_import
    CONF_PROFILE_ON: user_input.get(CONF_PROFILE_ON, DEFAULT_PROFILE_ON),
NameError: name 'user_input' is not defined
```
config looks like this:

```
fritzbox_tools:
  host: "192.168.1.3"
  username: !secret fritz_username
  password: !secret fritz_password
  devices:
    - "test-device"
  profile_on: "Standard"
  profile_off: "Gesperrt"
```
Any idea?
Regards
Lukas

**Comment by AaronDavidSchneider** [comment]
Thanks for your report @lweberru.
This bug is know and will be fixed with the merge of #59 

In the meantime you can bypass this problem by configuring the integration from the integration panel.

**Comment by mammuth** [comment]
Actually we should probably fix the issue before releasing the mentioned PR, since this won't happen until HA 0.105 is released.
